Dockerfile.armhf-alpine

```diff
@@ -0,0 +1,41 @@
+FROM resin/armhf-alpine-node:0.10.44-slim
+
+# Supervisor apt dependencies
+RUN apk add --update \
+		btrfs-progs \
+		ca-certificates \
+		rsync \
+		supervisor \
+	&& rm -rf /var/cache/apk/*
+
+# Copy supervisord configuration files
+COPY config/supervisor/ /etc/supervisor/
+
+# Install dependencies
+WORKDIR /app
+COPY package.json postinstall.sh /app/
+RUN set -x \
+	&& buildDeps='g++ sqlite-dev sqlite-libs make' \
+	&& apk add --update $buildDeps \
+	&& JOBS=MAX npm install --unsafe-perm --production --no-optional --build-from-source \
+	&& npm dedupe \
+	&& npm cache clean \
+	&& rm -rf /tmp/* \
+	&& apk del $buildDeps \
+	&& rm -rf /var/cache/apk/*
+
+# Copy source
+COPY . /app/
+
+RUN mv -f /app/entry.alpine.sh /app/entry.sh
```

Comment:
Can we instead change the `ENTRYPOINT` and `ln -sf /app/entry.sh /start` entries below to point to the alpine entry, rather than replacing the other